docs/DeveloperGuide.adoc

```diff
@@ -295,60 +295,263 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+|`* * *` |new user |See command instruction and usage |Refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |user |add a new record | Track my spending/income
 
-|`* * *` |user |delete a person |remove entries that I no longer need
+|`* * *` |Forgetful user | Add additional details through a description for each record | Better keep track of the details of an expense/income
 
-|`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
+|`* * *` |Visually inclined user |Attach an image to each record |Conveniently record additional details of the expense instead of typing it all out
 
-|`* *` |user |hide <<private-contact-detail,private contact details>> by default |minimize chance of someone else seeing them by accident
+|`* * *` |user |Delete a record |Remove expenses/income that I no longer need
 
-|`*` |user with many persons in the address book |sort persons by name |locate a person easily
+|`* * *` |Careless user |Edit an existing record |Make changes to any mistakes made
+
+|`* * *` |user |View a record | View additional information of an expense/income
+
+|`* * *` |user |Tag a record | Categorize and better manage my records
+
+|`* * *` |user |Specify a time stamp on expenses|Know when I spend my money
+
+|`* * *` |user | Sort and filter records by category or tag | Know how my spending/income is distributed
+
+|`* * *` |user | Archive past records | Better manage current expenses
+
+|`* * *` | user | Unarchive records |
```

Comment:
Would you consider having unarchive records as a must have, or should it be classified as nice to have?